qt_gui_cpp/include/qt_gui_cpp/exports.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2011, Dirk Thomas, TU Darmstadt

```

Comment:
Since the code was not written in 2011 and not by me please use 2019 and Microsoft (or whatever you prefer) here. Same below for the other added file.

Reply:
Updated. Because I have no intention to make any additional copyright\license disclaimer for newly added files, I chose to remove the whole copyright section. Let me know if it is inappropriate.

Reply:
With nothing being mentioned it is unclear how these new files are licenced. So the new files should still have the BSD license header. What is the problem with just adjusting the referenced line with the current year and your name / your companies name?

Reply:
After checking with MSFT OSS guidence, there is no problems to add the copyright, but it needs to be in a consistent form, which is `Copyright (c) Microsoft Corporation. All rights reserved.`. I added it back now.